lib/profiling/README

```diff
@@ -56,3 +86,53 @@ psy_time_step_mod:swlat_update_c    11  0.19000E+01   0.17272E+00  0.24E-02
 -----------------------------------------------------------------------------
 * corrected for systematic error
 =============================================================================
+
+
+Dr Hook
+-------
+This wrapper library interfaces with ECMWF's Dr Hook library. This library
+appears not to be available as open source on a public server. It provides more
+functionality than just profiling, see the Dr Hook documentations for details.
```

Comment:
s/documentations/documentation/ and is this documentation available publicly anywhere?

Reply:
Nope, you need to know the secret handshake.

Seriously: I have no idea how to get DrHook. I got a copy from one of the UKMO people when we discussed that we should support DrHook, and the comment was (from memory) that DrHook is not available on any public repo. Googling found:
https://software.ecmwf.int/wiki/download/attachments/19661682/drhook.pdf
But I don't know if this is a link we should be using, trying to play around with the URL resulted only in 'page not found' (e.g. use just up to 'download' or so), and searching on the wiki for hook only resulted in various pdf files.

So .... no idea :(

Reply:
The documentation (and the package itself) are not public so I'm happy to leave this.